doc/benchmark.dtd

```diff
@@ -15,6 +15,7 @@
 
 <!ELEMENT option (#PCDATA)>
 <!ELEMENT propertyfile (#PCDATA)>
+<!ELEMENT multiproperty (#PCDATA)>
```

Comment:
Yes, but it would be good if users could explicitly select the default behavior.

Also, are there checks that reject any invalid values for this property?